raiden/network/transport/matrix/transport.py

```diff
@@ -361,10 +377,13 @@ def start(
 
         self._inventory_rooms()
 
+        def on_success(greenlet):
```

Comment:
I think this is still not necessary, however it aided me during debugging because it clears the greenlets list